boards/arm/sam4l_ek/sam4l_ek_defconfig

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Can you enable ARM_MPU here? Or are you concerned about the amount of image RAM available on this platform?

Reply:
Done.